x-pack/test/plugin_functional/plugins/resolver_test/public/applications/resolver_test/index.tsx

```diff
@@ -4,119 +4,71 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import * as React from 'react';
+import { Router } from 'react-router-dom';
+
+import React from 'react';
 import ReactDOM from 'react-dom';
-import { AppMountParameters } from 'kibana/public';
-import { I18nProvider } from '@kbn/i18n/react';
-import { IEmbeddable } from 'src/plugins/embeddable/public';
-import { useEffect } from 'react';
+import { AppMountParameters, AppMountContext } from 'kibana/public';
+import { useMemo } from 'react';
 import styled from 'styled-components';
+import { I18nProvider } from '@kbn/i18n/react';
+import { KibanaContextProvider } from '../../../../../../../../src/plugins/kibana_react/public';
+import { DataAccessLayer } from '../../../../../../../plugins/security_solution/public/resolver/types';
+import {
+  ResolverReduxProvider,
+  resolverStoreFactory,
+  ResolverWithoutProviders,
+  resolverMockDataAccessLayerWithNoAncestorsTwoChildren,
+} from '../../../../../../../plugins/security_solution/public';
 
 /**
  * Render the Resolver Test app. Returns a cleanup function.
  */
-export function renderApp(
-  { element }: AppMountParameters,
-  embeddable: Promise<IEmbeddable | undefined>
-) {
+export function renderApp(context: AppMountContext, parameters: AppMountParameters) {
```

Comment:
This no longer needs embeddable stuff, instead take the standard app mount context and app mount parameters.